docs/sphinx/manual/building_hpx.rst

```diff
@@ -1357,3 +1357,21 @@ or Unix variants.
 .. include:: ../../generated/cmake_toolchains.rst
 
 .. include:: ../../generated/cmake_variables.rst
+
+
+CMAKE Variable
+==============
+
+This lists the CMAKE variables and how user can enable CMAKE variables during
+make.
+
+- `PLUGIN` : This component is a plugin-able library.
+- `TYPE` : The type can be: EXECUTABLE, LIBRARY or COMPONENT.
+- `INSTALL`: Generates a install rule for the target.
+- `HPX_SetOutputPaths.cmake`: If you want the libs in directory where you can
+  add_executable, you can use `*_OUTPUT_DIRECTORY` variable. To mimic HPX's
+  behavior, set ``include(HPX_SetOutputPaths``) after ``find_package(HPX)``.
+- `LD_LIBRARY_PATH` : In |hpx| libraries are in a path that isn't found by the
```

Comment:
`LD-LIBRARY_PATH` is not a variable related to cmake but only usable during execution of the application. It also is specific to Linux.

Reply:
Can you write what should be correct heading ref https://github.com/STEllAR-GROUP/hpx/issues/1258#issuecomment-55564243.